### PR TITLE
Rubocop: Update 1.28.2->1.50.0; requires Ruby 2.6

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

**Comment by bastelfreak** [comment]
I need to validate the rubocop.yml on a few modules before we can release this. some namespaces changed.

**Comment by bastelfreak** [comment]
Tests:
* https://github.com/voxpupuli/puppet-collectd/pull/1027

**Review by rwaffen — APPROVED** [review]

File: voxpupuli-test.gemspec [hunk 1]
```diff
@@ -31,9 +31,10 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'rspec-puppet-utils', '~> 3.4'
 
   # Rubocop
-  # newest versions that still support Ruby 2.5
-  s.add_runtime_dependency 'rubocop', '~> 1.28.2'
-  s.add_runtime_dependency 'rubocop-rspec', '~> 2.10.0'
+  # newest versions that still support Ruby 2.6
+  # jruby 9.3 in Puppetserver 7 is compatible with C Ruby 2.6
+  s.add_runtime_dependency 'rubocop', '~> 1.50.0'
```
Comment (rwaffen) [inline voxpupuli-test.gemspec:36]: hmm had a look: 1.56 is latest

Comment (bastelfreak) [reply voxpupuli-test.gemspec:36]: yes, but that doesn't work on Ruby 2.6.

Comment (rwaffen) [reply voxpupuli-test.gemspec:36]: ah okay. thats the thing.

**Review by jay7x — APPROVED** [review]

**Pull request merged**